server.py

```diff
@@ -186,130 +187,124 @@ def get(key):
     return asset
 
 
-@route('/api/v1/assets', method="GET")
-@auth_basic
-def api_assets():
-    with db.conn(settings['database']) as conn:
-        assets = assets_helper.read(conn)
-        return make_json_response(assets)
-
-
 # api view decorator. handles errors
-def api(view):
+def api_response(view):
     @wraps(view)
     def api_view(*args, **kwargs):
         try:
-            return make_json_response(view(*args, **kwargs))
-        except HTTPResponse:
```

Comment:
I have a little doubt about this fragment.
For the bottle was this code:
'except HTTPResponse:`
I don't quite understand its purpose and I don't know if it's necessary to do something similar for Flask